text/0000-apm-tools.md

```diff
@@ -0,0 +1,59 @@
+# Contribute APM Tools Buildpacks
+
+## Summary
+
+The VMware Tanzu team has maintained eight APM tools Cloud-Native buildpacks for a number of years as proprietary buildpacks. VMware would like to contribute these buildpacks to the Paketo project.
+
+The list of buildpacks to contribute is:
+
+- Apache Skywalking
+- AppDynamics
+- Aternity
+- Dynatrace
+- Elastic APM
+- JProfiler
+- New Relic
+- YourKit
+
+Each of these buildpacks includes support for Java, some also include support for Node.js, and a few include support for PHP. They are all written in such a way that adding additional language families is possible.
+
+The buildpacks to contribute are written in Go and utilize the libcnb and libpak libraries. They all use pipeline-builder for managing Github Actions. 
+
+Each buildpack functions in a similar fashion. It will install any required dependencies (agents, libraries, etc..). It may install a runtime exec.d helper to application configuration settings at runtime (usually from bindings). It will contribute any modifications to runtime configuration or start commands to ensure that the agent is loaded when the application launches. Details for each buildpack [can be found here](https://docs.vmware.com/en/VMware-Tanzu-Buildpacks/services/tanzu-buildpacks/GUID-partner-integrations-partner-integration-buildpacks.html).
+
+## Motivation
+
+VMware is vested in the buildpacks community and believes that a great way to grow the buildpacks community is to expand the set of available buildpacks. There have been requests for additional APM tooling support from the community and this will fill that need.
+
+## Detailed Explanation
+
+This RFC proposes the following changes:
+
+1. Create a new Paketo sub-team called "APM Tools". This team will be the owner for all eight of these buildpacks, as well as the existing APM related buildpacks: Azure Application Insights, Google Stackdriver, and Datadog.
+2. Since the APM tooling buildpacks function across language families, the "APM Tools" sub-team will be seeded with one maintainer from each language family team. This is to ensure that there is representation  and ownership from each language family.
+3. Create eight new repositories, one for each buildpack to be contributed. The APM Tools team will own these repositories. The repository names will follow the pattern `Paketo <technology> Buildpack`, ex: `Paketo Apache Skywalking Buildpack`, as is the existing convention. GitHub repository names will be the name of the technology, ex: `apache-skywalking`. Buildpack ID and image registry names will match the GitHub repository name, ex: `docker.io/paktobuildpacks/apache-skywalking`.
```

Comment:
Should we re-name these to conform with https://github.com/paketo-buildpacks/rfcs/pull/223 as we bring them in? Or should we bring them in with their current name and re-name them later? I see pros and cons to each approach.

Reply:
Either works for me too.